contracts/common/tokens/StandardToken.sol

```diff
@@ -0,0 +1,38 @@
+pragma solidity ^0.5.2;
```

Comment:
What is this file for?

Reply:
test token contract for ERC223 ContractReciver.